src/ape_accounts/_cli.py

```diff
@@ -86,17 +87,36 @@ def generate(cli_ctx, alias):
 
 
 # Different name because `import` is a keyword
-@cli.command(name="import", short_help="Add a new keyfile account by entering a private key")
+@cli.command(
+    name="import", short_help="Add a new keyfile account by entering a private key or seed phrase"
+)
+@click.option(
+    "--mnemonic", "import_from_mnemonic", help="Import a key from a mnemonic", is_flag=True
+)
+@click.option("--hdpath", "custom_hd_path", help="Input HD Path to use", is_flag=True)
 @non_existing_alias_argument()
 @ape_cli_context()
-def _import(cli_ctx, alias):
+def _import(cli_ctx, alias, import_from_mnemonic, custom_hd_path):
     path = _get_container().data_folder.joinpath(f"{alias}.json")
-    key = click.prompt("Enter Private Key", hide_input=True)
-    try:
-        account = EthAccount.from_key(to_bytes(hexstr=key))
-    except Exception as error:
-        cli_ctx.abort(f"Key can't be imported: {error}")
-        return
+    if import_from_mnemonic:
+        mnemonic = click.prompt("Enter mnemonic seed phrase", hide_input=True)
+        if not custom_hd_path:
+            custom_hd_path = ETHEREUM_DEFAULT_PATH
+        else:
+            custom_hd_path = click.prompt("Input HD Path to use", default=ETHEREUM_DEFAULT_PATH)
+        EthAccount.enable_unaudited_hdwallet_features()
+        try:
+            account = EthAccount.from_mnemonic(mnemonic, account_path=custom_hd_path)
```

Comment:
Gah, you know what? We should make this work the same as `ape-ledger` and `ape-trezor` where you get to select the account number in the CLI.. Wouldn't that be sweet? Then we can have `ape-trezor` and `ape-ledger` make use of those shared HD Path CLI tools and all would be balanced in the universe.

Reply:
I wonder if we can do this with a `--menu` flag to enable account import scripting without the menu, or menu driven account import with the menu?